recipes/sagemaker-tensorflow-container/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sagemaker-tensorflow-container" %}
+{% set version = "2.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f7a92a8bb0264379ac64d1de61706575d9659234e08b5008ba6fb9f86909598f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or (osx and py==37)]
```

Comment:
Why these?

Reply:
@isuruf multiple run dependencies do not build on windows - namely, `sagemaker-containers` - so this must skip windows (and fails on it). in the setup.py, it says it doesn't work on py37 - i got it work on Linux, but with osx it raised the error `[Errno 17] File exists: 'libopenblasp-r0.3.6.dylib'` that went away when osx skipped py37.

Reply:
That osx error could be something we have to fix in the openblas package. Can you reenable it?

Reply:
@isuruf re-enabled in most recent commit below